gulp/tasks/test.js

```diff
@@ -42,7 +42,6 @@ function testUnitScripts( cb ) {
 function testUnitServer() {
   spawn(
     'tox',
-    [ 'cfgov/core/tests' ],
```

Comment:
Modifies `gulp test` behavior to run all Python unit tests, not just the ones under `core.test`.